utils/auth/function.ts

```
@@ -59,26 +56,26 @@ export const updateUserMetadata = async (
+      console.error("OTP Error:", error);
       return { success: false, error };
     }
-    
+
     // Log success with response data
-    console.log('OTP request successful. Response:', data);
+    console.log("OTP request successful. Response:", data);
     return { success: true, data };
-    
+
     return { success: true };
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)